docs/section22/Rule22-20.md

```diff
@@ -0,0 +1,40 @@
+
+# CHW&CW - Rule 22-20  
+
+**Rule ID:** 22-20  
+**Rule Description:** The baseline heat rejection Loop temperature reset = Load Reset.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**Appendix G Section Reference:** Section G3.1.3.11 Heat Rejection (System 7, 8, 11, 12 and 13)  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11b, 12b, 13b.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:**  
+
+1. get_baseline_system_types()
+
+**Applicability Checks:**  
+
+- Get B-RMR system types: `baseline_hvac_system_dict = get_baseline_system_types(B-RMR)`
+
+  - Check if B-RMR is modeled with at least one air-side system that is Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11b, 12b, 13b, i.e. with air-side system served by chiller(s), continue to the next applicability check: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-7", "SYS-8", "SYS-11.1", "SYS-11.2", "SYS-12", "SYS-13", "SYS-7B", "SYS-8B", "SYS-11B", "SYS-12B", "SYS-13B"]): CHECK_RULE_LOGIC`
+
+  - Else, rule is not applicable to B-RMR: `else: RULE_NOT_APPLICABLE`
+
+## Rule Logic:  
+
+- For each heat rejection device in B_RMR: `for tower_b in B_RMR.RulesetModelInstance.heat_rejections:`
+
+  **Rule Assertion:**
+
+  - Case 1: If heat rejection temperature reset type is load reset: `if tower_b.reset_type == "LOAD_RESET": PASS`
+
```

Comment:
FluidLoopDesignAndControl has TemperatureResetOptions which can be 'Other'. However, I think should be more specific like 'SystemSetpoint' or 'LeavingWaterTemperature' and 'EnteringWaterTemperature' and for the baseline we would verify that it is 'LeavingWaterTemperature'